cloud_function_go/function.go

```diff
@@ -0,0 +1,48 @@
+// Package p contains an HTTP Cloud Function.
+package p
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/logzio/logzio-go"
```

Comment:
I think it better to use your own http request instead of using this package as this one is too robust ( internal queue.. )  for sending logs